task_10/Maxim.Shcherbakan/src/main/java/com/hillel/tasks/StringTasks.java

```diff
@@ -0,0 +1,52 @@
+package com.hillel.tasks;
+
+public class StringTasks {
+
+  public static void main(String[] args) {
+    StringTasks stringTasks = new StringTasks();
+    int result = stringTasks.binaryToDecimal("1100");
+    System.out.println(result);
+  }
+
+  public String replaceEmail(String email) {
+    return email.replace("@", "[ at ]").replace(".", "[ dot ]");
+  }
+
+  public String shortenWords(String sentence) {
+    String[] array = sentence.trim().split(" ");
+    for (int i = 0; i < array.length; i++) {
+      if (array[i].length() < 4) {
+        continue;
+      }
+      StringBuilder word = new StringBuilder();
+      array[i] = word
+          .append(array[i].charAt(0))
+          .append(array[i].length() - 2)
+          .append(array[i].charAt(array[i].length() - 1))
+          .toString();
+    }
+    return String.join(" ", array);
```

Comment:
It's the same as using a StringJoiner. Better to use it from the beginning instead of StringBuilder